Document/0x06b-Basic-Security-Testing.md

```diff
@@ -376,9 +374,76 @@ Save the IPA file locally with the following command:
+
+    - Open the app and leave it running on foreground.
+    - Start an objection session by running the following command:
+
+        ```shell
+        $ objection --gadget Telegram explore
+        Using USB device `iPhone`
+        ```
+
+    - Run `env` to display directory information for the current application environment. On iOS devices, this includes the location of the app's bundle (`BundlePath`), the Documents/ and Library/ directories.
+
+        ```shell
+        ph.telegra.Telegraph on (iPhone: 11.1.2) [usb] # env
+
+        Name               Path
+        -----------------  -------------------------------------------------------------------------
+        BundlePath         /var/containers/Bundle/Application/B0E38F10-8F30.../Telegram X.app
+        CachesDirectory    /var/mobile/Containers/Data/Application/56E142D2-D2CB.../Library/Caches
+        DocumentDirectory  /var/mobile/Containers/Data/Application/56E142D2-D2CB.../Documents
+        LibraryDirectory   /var/mobile/Containers/Data/Application/56E142D2-D2CB.../Library
+        ```
+
+    - `BundlePath` is also the current directory by default, run `ls` to list the contents:
+
+        ```shell
+        ph.telegra.Telegraph on (iPhone: 11.1.2) [usb] # ls
+
+        NSFileType      Perms  NSFileProtection   ... Size       Name
+        ------------  -------  ------------------ ... ---------  ----------------------------------
+        Directory         493  None               ... 224.0 B    PlugIns
+        Directory         493  None               ... 96.0 B     Base.lproj
+        Directory         493  None               ... 96.0 B     _CodeSignature
+        Directory         493  None               ... 1.3 KiB    Frameworks
+        ...
+        Regular           493  None               ... 1.4 MiB    Telegram X
+        ...
+        Readable: True  Writable: False
+        ```
+
+        The name of the app binary can be found in the `Info.plist` file by searching for the key `CFBundleExecutable` (running `ios plist cat Info.plist` will display the `Info.plist` file).
+    - Download the app binary using the command `file download`:
+
+        ```shell
+        ph.telegra.Telegraph on (iPhone: 11.1.2) [usb] # file download "Telegram X"
+
+        Downloading /var/containers/Bundle/Application/B0E38F10-8F30-4142-8C53-4CE022C2B097/
+            Telegram X.app/Telegram X to Telegram X
+        Streaming file from device...
+        Writing bytes to destination...
+        Successfully downloaded /var/containers/Bundle/Application/B0E38F10-8F30-4142-8C53-4CE022C2B097/
+            Telegram X.app/Telegram X to Telegram X
+        ```
+
+    Alternatively you can connect per SSH to the device, search for the bundle directory and `cd` to it, locate the app binary and copy it over to your computer (via SCP for example) or keep working on the device.
```

Comment:
SCP -> `scp`

https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#in-text-keywords